packages/fiori/test/specs/Page.spec.js

```diff
@@ -0,0 +1,26 @@
+const assert = require('chai').assert;
+
+describe("Page general interaction", () => {
+	browser.url("http://localhost:8081/test-resources/pages/Page.html");
+
+	it("tests initial rendering", () => {
+        const page = browser.$("#page");
+
+        assert.ok(page.isExisting(), "The component should have a shadow root.");
+        assert.ok(page.shadow$(".ui5-page-header-root"), "Header should be rendered.");
```

Comment:
This pattern below will be always truthy - isExisting() is the way to check if DOM element is rendered, like `page.isExisting()`.

```js
page.shadow$(".ui5-page-header-root")
```